### PR TITLE
update background image

### PR DESCRIPTION
**Pull request by rhod8215**
Trello Ticket

- [Update background image in sign up and sign in page](https://trello.com/c/K9EP6mh5/27-update-background-image-in-sign-up-and-sign-in-page)

Demo

- Before PR
Screen Shot 2019-10-03 at 10 38 24 AM
Screen Shot 2019-10-03 at 10 38 03 AM

- After PR

Screen Shot 2019-10-03 at 1 06 26 PM
Screen Shot 2019-10-03 at 1 06 50 PM
Screen Shot 2019-10-03 at 1 07 28 PM
Screen Shot 2019-10-03 at 1 07 51 PM
Screen Shot 2019-10-03 at 1 08 07 PM
Screen Shot 2019-10-03 at 1 10 32 PM

### CODE REVIEW

**Comment by jadurani** [comment]
Demo links are not working

**Comment by jadurani** [comment]
@rhod8215 
Next time include screenshots like the following or at least make sure the demo links are working

Screenshot from 2019-10-02 12-22-09
Screenshot from 2019-10-02 12-21-47
Screenshot from 2019-10-02 12-21-31

File: package.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
     "karma-coverage-istanbul-reporter": "~2.0.1",
     "karma-jasmine": "~2.0.1",
     "karma-jasmine-html-reporter": "^1.4.0",
+    "node-sass": "^4.12.0",
```
Comment (jadurani) [inline package.json]: what do you need this for

Comment (rhod8215) [reply package.json]: i applied the suggested solution/s from stackoverflow when i got the error after the src url of image changed.

ERROR in ./src/theme/variables.scss (./node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/raw-css-loader.js!./node_modules/postcss-loader/src??embedded!./node_modules/sass-loader/lib/loader.js??ref--15-3!./src/theme/variables.scss) Module Error (from ./node_modules/postcss-loader/src/index.js):

**Comment by jadurani** [comment]
@rhod8215 are the screenshots in the PRs description from the latest update? why are the input fields still white when they're supposed to already be transparent? The toolbars are still white

**Pull request merged**